core/graph/src/main/java/uk/gov/gchq/gaffer/graph/Graph.java

```diff
@@ -480,6 +482,38 @@ public Builder storeProperties(final URI propertiesURI) {
             return this;
         }
 
+        public Builder addStoreProperties(final StoreProperties updateProperties) {
+            if (this.properties == null) {
```

Comment:
Are we sticking to a convention of ordering the constant part of the equality check first?

Reply:
Change it I will.
More of them is there any?

Reply:
Unnatural it feels,
Change it I will,
More of them is there any?